docs/guides/universal-receiver-delegate/accept-reject-assets.md

```diff
@@ -0,0 +1,213 @@
+---
+sidebar_label: 'Accept & Reject Assets'
+sidebar_position: 2
+---
+
+# Accept & Reject Assets
+
+Each user can create its own **custom Universal Receiver Delegate** contract that holds its own logic to be executed once the **[`universalReceiver(..)`](../../standards/smart-contracts/lsp0-erc725-account.md#universalreceiver)** function on his profile is called. 
+
+![LSP1UniversalReceiverDelegate-Guide](/img/guides/UniversalReceiverDelegate-Guide.jpeg)
```

Comment:
I think diagrams look cooler when using emojis instead of these kind of old school stock icons :)

⚙️

Reply:
Hmm, I see your point, I guess we can keep it for now (for consistency), and later we can switch all figures to more modern use of emojis.